src/ru/danikirillov/App.java

```diff
@@ -23,9 +23,48 @@ private static void changeWheelNoErrorsTest()
     //Third, check results
     if (car.isAnyWheelFlat()) {
       System.out.println("changeWheelNoErrorsTest - ERROR!!!!!!!!!!! - wheel hasn't been changed");
-    } else {
+    }
+    else {
       System.out.println("changeWheelNoErrorsTest - SUCCESS");
     }
   }
 
+  public static void changeGoodWheelsNoErrorTest() {
+    final var radius = 17;
+    final var car = CarFactory.createGoodCar(radius);
+    final var wheel = new Wheel(false, radius);
+    car.replaceFlatWheel(wheel);
+    if (car.isAnyWheelFlat()) {
+      System.out.println("changeGoodWheelsNoErrorTest - ERROR!!!!!!!!!!! - wheel hasn't been changed");
```

Comment:
as i said in the line 12 - logically each unit test contains 3 parts. So put blank lines between them - that will make it much easier for other developers to understand your code.

```suggestion
    final var wheel = new Wheel(false, radius);
    
    car.replaceFlatWheel(wheel);
    
    if (car.isAnyWheelFlat()) {
      System.out.println("changeGoodWheelsNoErrorTest - ERROR!!!!!!!!!!! - wheel hasn't been changed");
```

Reply:
the same for other tests